index.js

```diff
@@ -2,20 +2,26 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Scott Beck @bline
 */
+
 var loaderUtils = require("loader-utils");
+
 module.exports = function(source) {
-	this.cacheable && this.cacheable(true);
-	var jade = require("jade");
+	this.cacheable && this.cacheable();
+
+	var jade  = require("jade");
 	var query = loaderUtils.parseQuery(this.query);
-	var req = loaderUtils.getRemainingRequest(this).replace(/^!/, "");
+	var req   = loaderUtils.getRemainingRequest(this).replace(/^!/, "");
+
 	var tmplFunc = jade.compile(source, {
-		filename: req,
-		self: query.self,
-		pretty: query.pretty,
-		locals: query.locals,
-		doctype: query.doctype || 'html',
+		filename:     req,
```

Comment:
You should not change the coding style of others modules if you want to get this merged easy